api/CorrelationContext/Correlations.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\CorrelationContext;
+
+interface Correlations
+{
+    /*******************************************************************************************************************
+     *
+     * This is following the correlation context API as defined here:
+     * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/correlationcontext/api.md
+     *
+     ******************************************************************************************************************/
+
+    /*
+    * Returns the name/value pairs in the CorrelationContext.
+    * The order of name/value pairs MUST NOT be significant.
+    * Based on the language specification, the returned value can be either an immutable collection
+    * or an immutable iterator to the collection of name/value pairs in the CorrelationContext.
+    * OPTIONAL parameters:
+    * Context: the context containing the CorrelationContext from which to get the correlations.
+    *
+    * @param ?array $Context
+    * @return array Returns the name/value pairs in the CorrelationContext.
+    */
+    public function getCorrelations(?array $Context): array;
```

Comment:
I think that we must define a new API for Context and use that API here. I don't think that the Context can be modeled properly using an array, because the specs say that the Context should be immutable and any change on a Context should result in a new Context.

Reply:
Agreed with you; I started implementing this and then started reading through the context specification.  I'll change this to use the context definition after we've created one.

Reply:
Also, the specs say that the getCorrelations method may receive a Context and if so, then it must return the  Correlations tied to that Context.

Reply:
Will change that as well when the time comes.